hydrogym/firedrake/envs/cavity/flow.py

```diff
@@ -11,6 +11,18 @@
 
 
 class Cavity(FlowConfig):
+  # Velocity probes
```

Comment:
probs are used now and the old setting of _A. Barbagallo, D. Sipp, P. J. Schmid - Closed-loop control of an open cavity flow using  reduced-order models (2014)_ is abandoned?

Reply:
Looks like it's still supported (as I think it should be), but not the default measurement.

Reply:
Indeed, probes can be set optionally and the old setting can still be used.